RELEASE_FLOW.md

```diff
@@ -35,27 +34,26 @@ use them on production environment. They also might not be documented.
 
 ### Creating enhancement/bugfix
 
-Branch from **rc** or **release-{version}** branch (if this is a bugfix for a scheduled release), code, merge back to
-the branch that you used in the first place.
+Branch from **master** branch, code, merge back to the branch that you used in the first place.
```

Comment:
in the first place = **master**, right? then "Branch from **master** branch, code, merge back when feature is ready."